Build/update-references.sh

```diff
@@ -8,6 +8,7 @@ BRANCH=$(echo ${payload} | jq --raw-output '.ref | match("refs/heads/(.+)") | .c
 # reset distribution
 git reset --hard
 git checkout -B ${BRANCH} origin/${BRANCH}
+git reset --hard origin/${BRANCH}
```

Comment:
So is the previous hard reset even needed any more? I guess it doesn't hurt and helps make relatively sure the checkout works?